container/Dockerfile

```diff
@@ -37,19 +48,19 @@ RUN echo -e 'startup_message off \nhardstatus on \nhardstatus alwayslastline \nv
 # Setup vim 
 RUN echo -e 'syntax enable \nset tabstop=4 \nset number \nset hlsearch \nset incsearch' > ${HOME}/.vimrc
 
-# Setup spark/scala/python/rest
-ENV SPARK_HOME="/spark_home"
-ENV PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin
+# Setup java/scala/python/kafka/rest
```

Comment:
Will you delete all of the below ?  At this point it makes sense for all the "setup" to be driven by `setup-container.sh` so it's in a single place (whether versioning applies or not) ?